src/main/resources/application.yaml

```diff
@@ -1,6 +1,7 @@
 spring:
   datasource:
-    url: 'jdbc:h2:file:./target/database/h2'
+    #url: 'jdbc:h2:file:./target/database/h2'
+    url: jdbc:h2:mem:test_mem
```

Comment:
I'm not sure I love the default behaviour being a memory DB.. Feels like this will result in confused newbies.

Reply:
I can see what you mean. It's fixed now

Reply:
What about the other defaults? I've out commented the swiss one, but kept IPS as an example. Would that be ok for you?

Reply:
@jvitrifork IPS feels like a sensible example for sure - although probably it should be commented out by default

Reply:
Done